api/src/pcapi/core/users/factories.py

```diff
@@ -145,37 +164,91 @@ def beneficiaryFraudChecks(  # pylint: disable=no-self-argument
+    idPieceNumber = factory.Faker("ssn", locale="fr_FR")
 
-        # Profile completion
-        profile_completion_fraud_check = fraud_factories.ProfileCompletionFraudCheckFactory(user=obj)
+    @classmethod
+    def beneficiary_fraud_checks(
+        cls, obj: models.User, **kwargs: typing.Any
+    ) -> list[fraud_models.BeneficiaryFraudCheck]:
+        import pcapi.core.fraud.factories as fraud_factories
 
-        # Identity validation
+        fraud_checks = super().beneficiary_fraud_checks(obj, **kwargs)
         identity_check_type = kwargs.get("type", fraud_models.FraudCheckType.UBBLE)
         identity_fraud_check = fraud_factories.BeneficiaryFraudCheckFactory(
             user=obj,
             type=identity_check_type,
             status=fraud_models.FraudCheckStatus.OK,
         )
+        fraud_checks.append(identity_fraud_check)
+        return fraud_checks
+
+    @factory.post_generation
+    def beneficiaryFraudChecks(  # pylint: disable=no-self-argument
+        obj,
+        create: bool,
+        extracted: fraud_models.BeneficiaryFraudCheck | None,
+        **kwargs: typing.Any,
+    ) -> list[fraud_models.BeneficiaryFraudCheck]:
+        if not create:
+            return []
+
+        return IdentityValidatedUserFactory.beneficiary_fraud_checks(obj, **kwargs)
 
-        # Honor statement
-        honor_statement_fraud_check = fraud_factories.HonorStatementFraudCheckFactory(user=obj)
 
-        fraud_checks = [
-            phone_validation_fraud_check,
-            profile_completion_fraud_check,
-            identity_fraud_check,
-            honor_statement_fraud_check,
-        ]
+class HonorStatementValidatedUserFactory(IdentityValidatedUserFactory):
+    @classmethod
+    def beneficiary_fraud_checks(
+        cls, obj: models.User, **kwargs: typing.Any
+    ) -> list[fraud_models.BeneficiaryFraudCheck]:
+        import pcapi.core.fraud.factories as fraud_factories
 
+        fraud_checks = super().beneficiary_fraud_checks(obj, **kwargs)
+        fraud_checks.append(fraud_factories.HonorStatementFraudCheckFactory(user=obj))
         return fraud_checks
 
+    @factory.post_generation
+    def beneficiaryFraudChecks(  # pylint: disable=no-self-argument
+        obj,
+        create: bool,
+        extracted: fraud_models.BeneficiaryFraudCheck | None,
+        **kwargs: typing.Any,
+    ) -> list[fraud_models.BeneficiaryFraudCheck]:
+        if not create:
+            return []
+
+        return HonorStatementValidatedUserFactory.beneficiary_fraud_checks(obj, **kwargs)
+
+
+class BeneficiaryFactory(HonorStatementValidatedUserFactory):
+    """
+    Generates a beneficiary as if it had finished the deposit activation process.
+    On top of the base user form, the user went through
+    - email validation
+    - phone number validation
+    - profile completion
+    - identity validation
+    - honor statement
+    - deposit activation (automatic after all the above)
+    """
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏J'aime bien c'est assez clair ces factories imbriquées !

<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Cette docstring, on pourrait aussi l'avoir sur les autres factories, qu'en penses tu ? (low logaf)